Library/Formula/fzf.rb

```diff
@@ -1,10 +1,10 @@
 require "language/go"
 
 class Fzf < Formula
-  desc "Fuzzy finder for your shell"
+  desc "A command-line fuzzy finder written in Go"
```

Comment:
Please remove the indefinite article at the beginning: `desc "Command-line ...` instead of `desc "A command-line ...`.